src/riak_kv_pb_timeseries.erl

```diff
@@ -127,6 +130,53 @@ process(#tsputreq{table = Bucket, columns = _Columns, rows = Rows}, State) ->
             {reply, missing_helper_module(Bucket, BucketProps), State}
     end;
 
+%% @ignore DELETE (as if)
+process(#tsdelreq{table = Table, key = CompoundKey_,
+                  vclock = VClock_, timeout = Timeout},
+        State) ->
+    Options =
```

Comment:
I think `lists:flatmap` is overkill when you only have a singleton list you're processing :). This could be made much simpler and clearer by just using a case statement.